src/main/webapp/app/overview/course-conversations/layout/conversation-selection-sidebar/conversation-selection-sidebar.component.html

```diff
@@ -40,9 +40,10 @@ <h3 class="card-title">
                             [activeConversation]="activeConversation"
                             (conversationSelected)="onConversationSelected($event)"
                             [course]="course"
-                            (settingsChanged)="onSettingsChanged()"
-                            (conversationHiddenStatusChange)="onConversationHiddenStatusChange()"
-                            (conversationFavoriteStatusChange)="onConversationFavoriteStatusChange()"
+                            (settingsDidChange)="onSettingsDidChange()"
```

Comment:
I think we should definitely refactor this file. But that would go beyond the scope of this PR.

Reply:
I documented it https://github.com/ls1intum/Artemis/issues/7995